File: VideoLocker/src/main/java/org/edx/mobile/discussion/CommentBody.java
package org.edx.mobile.discussion;

public class CommentBody {
    String threadId;
    String rawBody;
    String parentId;

    public void setThreadId(String threadId) {
        this.threadId = threadId;
    }

    public void setRawBody(String rawBody) {
        this.rawBody = rawBody;
    }

    public String getThreadId() {
        return threadId;
    }

    public String getRawBody() {
        return rawBody;
    }

    public void setParentId(String parentId) {
        this.parentId = parentId;
    }

    public String getParentId() {
        return parentId;
    }
}

File: OpenEdXMobile/src/main/java/org/edx/mobile/model/discussion/CommentBody.kt
package org.edx.mobile.model.discussion

import com.google.gson.annotations.SerializedName

data class CommentBody(
    @SerializedName("thread_id")
    val threadId: String,

    @SerializedName("raw_body")
    val rawBody: String,

    @SerializedName("parent_id")
    val parentId: String?
)
